requirements.txt

```diff
@@ -22,7 +22,7 @@ orjson==3.8.6
 pip>=21.0,<23.1
 python-slugify==4.0.1
 pyyaml==6.0
-requests==2.28.1
+requests==2.28.2
```

Comment:
`pyproject.toml`and `package_constraints.txt` also need to be updated.

Reply:
Added the bump in these files as well :)

Reply:
Added the compare view url in the description.